### ISSUE TITLE
Compatability with Comforts

### ISSUE THREAD

**Issue by valchiara** [CosmicDan-Minecraft/SleepingOverhaul2#1]
It would be nice if Sleeping Overhaul 2 worked with Comforts' sleeping bags

**Comment by cosmicdan** [comment]
Thanks for reporting, already solved and will come in next update (better late than never hehe)

**Comment by cosmicdan** [comment]
Hi,

There is no issue with Sleeping Bags from Comforts on the 1.20.1 version. Can you please confirm if the issue persists? Or if you are using a different Forge version?

**Comment by valchiara** [comment]
I think I might have mixed up sleeping bags and hammocks, sleeping bags do indeed work, however hammocks do not let you sleep during the day anymore with Sleeping Overhaul 2

**Comment by cosmicdan** [comment]
Indeed, this was my findings. I'll figure it out, even if it just means a quick-fix to let them skip to night for now.

⁣Get BlueMail for Android ​

**Comment by cosmicdan** [comment]
Solved in latest version
